_posts/2017-04-02-Increasing-Rusts-Reach-2.0.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Increasing Rust’s Reach 2.0"
```

Comment:
Could alternatively be "2018" rather than "2.0" to align with the "2018 Edition".

Reply:
works for me!

Reply:
zomg thank u i would have published this under the wrong year if you had not said something 😬 😅